package/network/utils/ipset/Makefile

```diff
@@ -53,8 +58,12 @@ endef
 define Package/ipset/install
 	$(INSTALL_DIR) $(1)/usr/sbin
 	$(CP) $(PKG_INSTALL_DIR)/usr/sbin/ipset $(1)/usr/sbin/
+endef
+
+define Package/libipset/install
 	$(INSTALL_DIR) $(1)/usr/lib
 	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libipset*.so* $(1)/usr/lib/
 endef
 
+$(eval $(call BuildPackage,libipset))
```

Comment:
Minor nit, but can you put this new entry below the ipset one, so it stays alphabetical, and in the same order of Package/*/install?